### PR TITLE
Patch up claims view

### PR DESCRIPTION
**Pull request by palmenhq**
# Jira Issue: []

## What?
- Report errors on claims page
- Sort payments by date
- Sort member claims by date
- Allow spaces and line breaks in claim notes
- Don't warn about on reserves on closed claims

## Why?
- UX

## Optional checklist
- [x] Updated `src/changelog.ts`
- [ ] Codescouted
- [ ] Unit tests written
- [x] Tested locally

### CODE REVIEW

File: shared/hedvig-ui/loadable.tsx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import { css, keyframes } from '@emotion/react'
 import styled from '@emotion/styled'
 import React from 'react'
-import { range } from 'utils/array'
```
Comment (palmenhq) [inline shared/hedvig-ui/loadable.tsx:4]: I realized you can't import any other alias package in `hedvig-ui`

File: src/components/claims/claim-details/components/ClaimNotes.tsx [hunk 2]
```diff
@@ -90,10 +92,12 @@ const ClaimNotes: React.FC<Props> = ({ claimId }) => {
     <Paper>
       <h3>Notes</h3>
       {loadingClaimNotes && <Spinner />}
+      {queryError && <ErrorText>{queryError.message}</ErrorText>}
+
       <MuiList>
         {sortNotesByDate(notes).map((note) => (
           <ListItem key={note.date + note.handlerReference}>
-            <ClaimNote component="p">{note.text}</ClaimNote>
+            <ClaimNote component="pre">{note.text}</ClaimNote>
```
Comment (vonElfvin) [inline src/components/claims/claim-details/components/ClaimNotes.tsx:100]: Neat

File: src/components/claims/claim-details/components/ClaimPayment.tsx [hunk 3]
```diff
@@ -130,7 +130,7 @@ const getPaymentValidationSchema = (isPotentiallySanctioned: boolean) =>
     exGratia: yup.boolean(),
     type: yup
       .string()
-      .oneOf(['Manual', 'Automatic', 'IndemnityCost', 'Expense'])
```
Comment (vonElfvin) [inline src/components/claims/claim-details/components/ClaimPayment.tsx:133]: Any particular reason we remove Manual? 🤔

File: src/components/claims/claim-details/index.tsx [hunk 4]
```diff
@@ -17,7 +17,10 @@ import { ClaimTypeForm } from './components/ClaimType'
 import { MemberInformation } from './components/MemberInformation'
 
 const GridWithChatPaneAdjustment = styled(Grid)`
-  @media (min-width: 1400px) {
+  max-width: 100%;
+  min-width: 1000px;
+
+  @media (min-width: 1700px) {
```
Comment (vonElfvin) [inline src/components/claims/claim-details/index.tsx:23]: I saw `clamp` being a neat thing to use in similar cases, maybe we should consider it!

Here is the video I saw it in if you want to watch that example: https://youtu.be/Qhaz36TZG5Y?t=263

File: src/graphql/use-get-member-claims.tsx [hunk 5]
```diff
@@ -17,6 +18,12 @@ export const useGetMemberClaims = (
       memberId,
     },
   })
-  const claims = queryResult?.data?.member?.claims as Claim[] | undefined
+  let claims = queryResult?.data?.member?.claims as Claim[] | undefined
```
Comment (vonElfvin) [inline src/graphql/use-get-member-claims.tsx:21]: I think we at some places do `| []` to default to an empty array, that is probably not the expected behaviour now that I think of it. We should probably do `|  undefined` as you do here

**Review by vonElfvin — APPROVED** [review]
Noice 💯

File: src/components/claims/claim-details/components/ClaimPayment.tsx (hunk 3 above)
Comment (palmenhq) [reply src/components/claims/claim-details/components/ClaimPayment.tsx:133]: Apparently we can't use it anymore for bookkeeping reasons

File: src/components/claims/claim-details/index.tsx (hunk 4 above)
Comment (palmenhq) [reply src/components/claims/claim-details/index.tsx:23]: That's so cool, never seen it before!

**Pull request merged**